Library/Homebrew/cask/cask.rb

```diff
@@ -129,19 +129,26 @@ def config_path
       metadata_main_container_path/"config.json"
     end
 
+    def checksumable?
+      url.using.nil? || url.using == :post
```

Comment:
I believe once #13232 we can change this to.
But until #13232 is merged this will result in an error.

@MikeMcQuaid Could we merge without support and include the changes in the other PR? I'm not sure on the status quo here.

```suggestion
      (url.using.nil? && url.only_paths.blank?) || url.using == :post
```

Reply:
For now it'll need to be
```suggestion
      url.using.blank? || url.using == :post
```